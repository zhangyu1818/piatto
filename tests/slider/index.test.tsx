import React from 'react'
import { Slider } from 'piatto'
import { render } from '@testing-library/react'
import mountTest from '../shared/mountTest'

describe('Input', () => {
  mountTest(Slider)

  it('should match snapshots', function () {
    const { baseElement } = render(<Slider min={0} max={100} step={1} value={50} />)
    expect(baseElement).toMatchSnapshot()
  })
})
