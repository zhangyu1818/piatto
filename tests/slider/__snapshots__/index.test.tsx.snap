// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should match snapshots 1`] = `
<body>
  <div>
    <div
      aria-label="Slider"
      class="piatto-slider"
      style="background-position-x: 0px;"
    >
      <div
        aria-label="Slider Handle"
        class="piatto-slider-handle"
        role="button"
        style="transform: translate(0px,-50%);"
      />
    </div>
  </div>
</body>
`;
